src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy

```diff
@@ -393,6 +393,31 @@ class RevapiSpec extends IntegrationSpec {
         assert stderr.contains('method void foo.Foo::setSomeProperty(java.lang.String)')
     }
 
+    def 'does not throw exception when baseline-circleci is applied before this plugin'() {
```

Comment:
I think the situation in my case was this was failing in a subproject. Baseline was applied at the root project as usual.

Reply:
The `baseline-circleci` does stuff with `getTasks()` on `allprojects`, so I think the effect is the same.

Reply:
I've made the test use a subproject now